package.json

```diff
@@ -57,11 +57,11 @@
     "build_es6": "npm-run-all clean_dist_es6 copy_src_es6 compile_module_es6",
     "build_es6_for_docs": "npm-run-all clean_dist_es6 copy_src_es6 compile_dist_es6_for_docs",
     "build_closure_core": "node ./tools/make-closure-core.js",
-    "build_global": "npm-run-all clean_dist_global build_es6 && mkdirp ./dist/global && node ./tools/make-umd-bundle.js && npm-run-all build_closure_core",
+    "build_global": "npm-run-all clean_dist_global build_es6 && mkdirp ./dist/global && node ./tools/make-umd-bundle.js && npm-run-all build_closure_core clean_dist_es6",
     "build_perf": "webdriver-manager update && npm-run-all build_cjs build_global perf",
     "build_test": "shx rm -rf ./dist/ && npm-run-all build_cjs clean_spec build_spec test_mocha",
     "build_cover": "shx rm -rf ./dist/ && npm-run-all build_cjs build_spec cover",
-    "build_docs": "npm-run-all build_global build_es6_for_docs build_cjs clean_spec build_spec tests2png decision_tree_widget && esdoc -c esdoc.json",
+    "build_docs": "npm-run-all build_global build_es6_for_docs build_cjs clean_spec build_spec decision_tree_widget && esdoc -c esdoc.json && npm-run-all clean_dist_es6",
```

Comment:
should tests2png have been removed? It's no longer used in any other script and I think it is indeed needed as part of the doc building process?